composer.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "ezsystems/ezplatform-i18n",
+  "description": "eZ Platform translations",
```

Comment:
@bdunogier no this is for developement only if you want to add new translation strings to the source folder.

There is more details in the readme.

I'll update to : "Meta repository to store every eZ Platform translations. This should be used during development to ease synchronization with Crowdin. SHOULD NOT BE USED IN PRODUCTION",